src/plugins/content_management/public/constants.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// central place for all content ids rendered by content management
+
+// page ids
+export const ESSENTIAL_OVERVIEW_PAGE_ID = 'analytics_overview';
```

Comment:
I am thinking we should import the DEFAULT_NAV_GROUPS from core and construct the page id by using `DEFAULT_NAV_GROUPS.essentials.id`, so we can make sure that the id in default_nav_groups.ts will be the source of truth.

Reply:
when i try this, the enum not support this.

![image](https://github.com/user-attachments/assets/10c9775e-eed2-4309-867b-eebe82ef178d)

Reply:
thanks, changed to
```typescript
export const ANALYTICS_ALL_OVERVIEW_PAGE_ID = `${ALL_USE_CASE_ID}_overview`;
export const ESSENTIAL_OVERVIEW_PAGE_ID = `${ESSENTIAL_USE_CASE_ID}_overview`;
export const SEARCH_OVERVIEW_PAGE_ID = `${SEARCH_USE_CASE_ID}_overview`;
```